build/build.go

```diff
@@ -129,3 +133,72 @@ func configureScript(script *exec.Cmd, extension core.Extension) error {
 }
 
 const rwxr_xr_x = 0755
+
+func setLocalization(extension *core.Extension) error {
+	localization, err := api.GetLocalization(extension)
```

Comment:
Can we move both `GetLocalization` and `setLocalization` to its own file or just make it part of `build.go`? It doesn't really belong in `api.go` now.

Reply:
Just got some extra context and looks like it can be removed 👍